exporter/exporterhelper/internal/persistent_queue_test.go

```diff
@@ -167,20 +168,20 @@ func TestPersistentQueue_ConsumersProducers(t *testing.T) {
 	}
 }
 
-func newTraces(numTraces int, numSpans int) pdata.Traces {
-	traces := pdata.NewTraces()
+func newTraces(numTraces int, numSpans int) ptrace.Traces {
+	traces := ptrace.NewTraces()
 	batch := traces.ResourceSpans().AppendEmpty()
 	batch.Resource().Attributes().InsertString("resource-attr", "some-resource")
 	batch.Resource().Attributes().InsertInt("num-traces", int64(numTraces))
 	batch.Resource().Attributes().InsertInt("num-spans", int64(numSpans))
 
 	for i := 0; i < numTraces; i++ {
-		traceID := pdata.NewTraceID([16]byte{1, 2, 3, byte(i)})
+		traceID := pcommon.NewTraceID([16]byte{1, 2, 3, byte(i)})
```

Comment:
I can see why NewTraceID and NewSpanID are in `pcommon` as they're used by multiple signals, but since they're always tied to the tracing signal, would it not make more sense for them to live in `ptrace`?

Reply:
I'd put them in pcommon just to maintain a clean dependancy chain and avoid any dependancies across the signal packages.

Reply:
Crazy consideration/idea. Can we remove them and replace with a `[8]byte` and `[16]byte` for the APIs in span/exemplar/etc.?

Reply:
This can be a followup issue to discuss.

Reply:
Sure, submitted https://github.com/open-telemetry/opentelemetry-collector/issues/5137